demo/client/src/view.ts

```diff
@@ -197,6 +202,60 @@ export abstract class DemoView extends PathfinderView implements RenderContext {
 
         this.atlasRenderingTimerQuery = null;
         this.compositingTimerQuery = null;
+
+        this.inVRRAF = false;
+        this.inVR = false;
+        this.vrDisplay = null;
+        if ("VRFrameData" in window) {
+           this.vrFrameData = new VRFrameData;
+        } else {
+            this.vrFrameData = null;
+        }
+
+        this.vrSetup();
+    }
+
+    vrSetup(): void {
+        if (navigator.getVRDisplays) {
```

Comment:
nit: Let's use `if ('getVRDisplays' in navigator) { ... }` for consistency with the above.